cluster/gce/config-common.sh

```diff
@@ -27,7 +27,10 @@ function get-num-nodes {
 #   NUM_NODES
 #   NUM_WINDOWS_NODES
 function get-master-size {
-  local suggested_master_size=2
```

Comment:
This needs to be different. We are running additional controller on master machine

Reply:
Added setting on presubmit level, no need to revert this anymore

See kubernetes/test-infra#26662

Reply:
Actually, it's still needed (and also node size override to n1-standard-4), as kubetest2 does not support presets.